config/config_test.go

```diff
@@ -569,6 +656,35 @@ func TestInvalidSamplerType(t *testing.T) {
 	rcs.Close()
 }
 
+func TestInvalidRemoteSamplerStrategySettings(t *testing.T) {
```

Comment:
isn't this already tested in TestInvalidSamplerConfig?

Reply:
No that doesn't use `cfg.NewSampler`, it uses `cfg.samplerConfigFromEnv` which are different code paths with different errors.